### PR TITLE
Add CDNJS version integration

### PR DESCRIPTION
**Pull request by PeterDaveHello**
To show the latest(and usually the stable) version on CDNJS via CDNJS api.

cdnjs-jquer-version-badge-example

cc cdnjs/cdnjs#5060

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@FredyC thanks for your review :+1:

**Comment by PeterDaveHello** [comment]
If it's possible please also help review #767 #768 , thank you very much :smile:

File: server.js [hunk 1]
```diff
@@ -1408,6 +1408,32 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CDNJS version integration
+camp.route(/^\/cdnjs\/v\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var library = encodeURIComponent(match[1]);  // eg, "express" or "@user/express"
+  var format = match[2];
+  var apiUrl = 'https://api.cdnjs.com/libraries/' + library + '?fields=version';
+  var badgeData = getBadgeData('CDNJS', data);
```
Comment (espadrine) [inline server.js]: Could you put `cndjs` in lowercase please? The left-hand side is usually all-lowercase.

Comment (PeterDaveHello) [inline server.js]: Updated! BTW, is it possible to add link to the generated markdown code by default? Like to link to the library on CDNJS, thanks.

Comment (PeterDaveHello) [inline server.js]: @espadrine Sorry that I didn't be clear enough, actually I mean to dynamic generate the link, is it possible right now? Thanks.

**Comment by PeterDaveHello** [comment]
💥  https://twitter.com/PeterDaveHello/status/787625531990609920

File: server.js (hunk 1 above)
Comment (PeterDaveHello) [inline server.js]: @espadrine #838

**Pull request merged**